dronecore/plugin_manager.py

```diff
@@ -60,24 +59,19 @@ def _spinup_backend(self):
         """
         Spinup a backend and connect to it
         """
+        #: backend is running on localhost
+        self.port = "127.0.0.1"
```

Comment:
I'm not sure I understand the worgind here: default port is 50051, and address is localhost.

Reply:
That's indeed wrong. I haven't tested the function yet as I am not sure how we will ship the backend with the python library!

Reply:
Right. Well, the simple assumption is "the backend is already running on an \<IP:port\> somewhere". And on top of that, we can provide an interface that runs a backend locally. But it may be that the backend runs on a different machine (e.g. directly on the drone), so at least one should be able to connect to \<IP:port\>.

In Swift, that's currently the only interface (e.g. `let core = Core("localhost", 50051)`).

Reply:
That is possible? Have a look at ``__init__`

```
import dronecore

# Connect to a backend
remote_backend = dronecore.connect(host="192.168.178.56", port=50051)

# Start a backend local and connect
local_backend = dronecore.connect()
```

Reply:
yeah, I know :). I was just answering to your comment, but maybe I missed your point actually ;-).